tests/install-service

```diff
@@ -3,24 +3,25 @@
 set -eufx
 
 SECRETS=/var/lib/cockpit-tests/secrets
-IMAGES=/var/lib/cockpit-tests/images
+CACHE=/var/cache/cockpit-tests
 
-mkdir -p $SECRETS $IMAGES
-chown -R 1111:1111 $SECRETS $IMAGES
+mkdir -p $SECRETS $CACHE
+chown -R 1111:1111 $SECRETS $CACHE
 
 cat <<EOF > /etc/systemd/system/cockpit-tests.service
 [Unit]
 Description=Cockpit Verify
-Requires=docker.service libvirtd.service
-After=docker.service libvirtd.service
+Requires=docker.service
+After=docker.service
```

Comment:
Ah, nice cleanup along the way, now the libvirt stuff is happening inside the docker container (through vm-prep).